src/main/java/com/speaker/service/KafkaHandlerErrorsService.java

```diff
@@ -0,0 +1,5 @@
+package com.speaker.service;
+
+public interface KafkaHandlerErrorsService<T>{
```

Comment:
Looks like `T` must be as `BaseEntityDTO`

Reply:
OK. Change made